### PR TITLE
38 - Update to Polymer 2.x - part 5

### PR DESCRIPTION
**Pull request by dotpointer**
Update to Polymer 2.x - part 5.

* Replacing Polymer.dom.

Issue is #38.

### CODE REVIEW

**Comment by dotpointer** [comment]
Tested this on frontend.

I got the following in the console:

```
Uncaught TypeError: Cannot set property 'selected' of null
    at HTMLElement._changeRoute (cosmoz-page-router.js:598)
    at HTMLElement._activateCustomElement (cosmoz-page-router.js:532)
    at HTMLElement._activateImport (cosmoz-page-router.js:463)
    at HTMLLinkElement.importLoadedCallback (cosmoz-page-router.js:401)
```

So, something is breaking with this pull request.

**Comment by dotpointer** [comment]
Reassigning to you @megheaiulian as discussed on Slack.

File: cosmoz-page-router.js [hunk 1]
```diff
@@ -121,6 +135,7 @@
 		}
 
 		ready() {
+			super.ready();
```
Comment (megheaiulian) [inline cosmoz-page-router.js]: @dotpointer This here is the reason why `cosmoz-page-router` wasn't even initialized as an element.

Comment (dotpointer) [reply cosmoz-page-router.js]: Ok, thanks for the information.

**Pull request merged**